src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheck.java

```diff
@@ -125,6 +125,28 @@
  * <pre>
  *  &lt;module name=&quot;HiddenField&quot;/&gt;
  * </pre>
+ * <pre>
+ * public class SomeClass {
+ *
+ *   private String field; // field 1
+ *   private String test; // field 2
+ *
+ *   // Parameterized constructor
```

Comment:
please remove comments, our users are knowledgeable on basics.
please do this for other lines.
Please keep only trailing comments that explains behavior of Check

Reply:
done